paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentMethodsUiExtension.kt

```diff
@@ -170,7 +151,19 @@ private fun CardBrand.getDayIcon(): Int {
 }
 
 @DrawableRes
-internal fun getLinkIcon(showNightIcon: Boolean? = null): Int {
+internal fun getLinkIcon(
+    showNightIcon: Boolean? = null,
+    iconOnly: Boolean = false,
+): Int {
+    if (iconOnly) {
+        return getOverridableIcon(
```

Comment:
You're not using any of the functionality of `getOverridableIcon`

Return the resource for the icon directly in that case.

```suggestion
       if (iconOnly) { return R.drawable.stripe_ic_paymentsheet_link_arrow}
```